examples/statistics/confidence_ellipse.py

```diff
@@ -214,3 +214,15 @@ def get_correlated_dataset(n, dependency, mu, scale):
 
 fig.subplots_adjust(hspace=0.25)
 plt.show()
+
+#############################################################################
+#
+# ------------
+#
+# References
+# """"""""""
+#
+# The use of the following functions and methods is shown in this example:
+
+import matplotlib
+matplotlib.transforms.Affine2D
```

Comment:
I'd add `matplotlib.patches.Ellipse`

Reply:
good point. Although it seems that the backlinking does not work for classes like `Ellipse` or `Affine2D`. See https://51381-1385122-gh.circle-artifacts.com/0/doc/build/html/api/transformations.html#matplotlib.transforms.Affine2D where this example is not linked to :(

Reply:
I added that the Ellipse in the latest push, though the issue with it rendering will remain I suppose